docs/Makefile

```diff
@@ -5,14 +5,19 @@
 DIAGRAMS := boot-1 boot-2 \
     build kernel debian_minimal_builder \
 
+TOP_DIR ?= $(abspath ..)
+
 all: $(addsuffix .svg,$(DIAGRAMS))
 all: $(addsuffix .pdf,$(DIAGRAMS))
+	mkdir -p $(TOP_DIR)/output-docs
+	cp *.dot *.html *.pdf *.svg $(TOP_DIR)/output-docs
```

Comment:
The dependancies here do not quite look right.

Reply:
I am not sure what you mean... the `$(TOP_DIR)`?